SpatialGDK/Documentation/content/get-started/gdk-and-starter-project.md

```diff
@@ -0,0 +1,70 @@
+# Get started: 3 - Set up the Unreal GDK module and Starter Project
+
+Follow the steps below to:
+
+* Clone the Starter Project repository.
+* Clone the Unreal GDK into the `Plugins` folder of your game.
+* Build the Unreal GDK module dependencies which the Starter Project needs so it can work with the GDK.
+* Run the Starter Project on SpatialOS
+
+### Cloning
+
+1. Clone the [Unreal GDK Starter Project](https://github.com/spatialos/UnrealGDKStarterProject/) repository by running either:
+    * (HTTPS) `git clone https://github.com/spatialos/UnrealGDKStarterProject.git`
+    * (SSH) `git clone git@github.com:spatialos/UnrealGDKStarterProject.git`
+1. Navigate into `<StarterProjectRepoRoot>/Game/Plugins`. You may need to create the Plugins folder if you don't have any plugins installed yet.
+1.  In a Git Bash terminal window, clone the [Unreal GDK](https://github.com/spatialos/UnrealGDK) repository by running either:
+    * (HTTPS) `git clone https://github.com/spatialos/UnrealGDK.git`
+    * (SSH) `git clone git@github.com:spatialos/UnrealGDK.git`
+
+**Note** You need to ensure that the root folder of the Unreal GDK repo is called UnrealGDK (`<StarterProjectRepoRoot>/Game/Plugins/UnrealGDK/...`)
```

Comment:
**Note** You need to ensure that the root folder of the Unreal GDK repository is called `UnrealGDK`, so its path is something like `<StarterProjectRepoRoot>\Game\Plugins\UnrealGDK\...`).

(Missing full stop and back ticks, "repo" instead of "repository", forward slashes for file paths, the thing in brackets isn't clear.)